specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-11-01/benefitAsyncApis.json

```diff
@@ -0,0 +1,910 @@
+      "get": {
+        "tags": [
+          "BenefitUtilizationSummariesAsync"
+        ],
+        "operationId": "BenefitUtilizationSummariesAsync_GetOperationStatusSavingsPlanScope",
+        "description": "Gets status of benefit utilization summaries report.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/cost-management/"
+        },
+        "x-ms-examples": {
+          "BenefitUtilizationSummariesOperationResultsBySavingsPlan": {
+            "$ref": "./examples/BenefitUtilizationSummariesOperationResultsBySavingsPlan.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/savingsPlanOrderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/savingsPlanIdParameter"
+          },
+          {
+            "$ref": "#/parameters/benefitsAsyncOperationIdParameter"
+          },
+          {
+            "$ref": "common-types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The status of the generation of the benefit utilization summaries report.",
+            "schema": {
+              "$ref": "#/definitions/BenefitUtilizationSummariesOperationStatus"
+            }
+          },
+          "202": {
+            "description": "Accepted. Request will be processed. Use the location header to check the status.",
+            "headers": {
+              "Location": {
+                "description": "The URL to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "format": "int32",
+                "type": "integer"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "BenefitUtilizationSummariesOperationStatus": {
+      "type": "object",
+      "description": "Status of a benefit utilization summaries report. Provides Async Benefit Utilization Summaries Request input, status, and report sas url.",
+      "properties": {
+        "input": {
+          "$ref": "#/definitions/AsyncBenefitUtilizationSummariesRequest",
+          "description": "Input given to create the benefit utilization summaries report."
+        },
+        "status": {
+          "description": "The status of the creation of the benefit utilization summaries report.",
+          "type": "string",
+          "enum": [
+            "Running",
+            "Complete",
```

Comment:
>Complete [](http://example.com/codeflow?start=13&length=8)

Succeeded is a standard verb that can be used instead

Reply:
This "Complete" verb is baked in to azure durable functions and is what will be returned.